reference-architecture/vmware-ansible/roles/nfs-server/tasks/main.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
@abutcher @sdodson any changes we can pull in to openshift-ansible that would allow for this to leverage shared config?